pkg/builder/container_builder_test.go

```diff
@@ -30,27 +30,27 @@ func TestGetName(t *testing.T) {
 		{
 			name:     "image name with registry hostname and tag",
 			image:    "gcr.io/my-repo/my-image:latest",
-			expected: "my-image",
+			expected: "my-repo-my-image",
```

Comment:
I think we should add a couple more test cases here, one to cover the case where there are no slashes in the image name, and one to cover the case were there are more than 2 slashes.